config/core.entity_form_display.node.training.default.yml

```
@@ -197,7 +197,7 @@ content:
     settings: {  }
     third_party_settings: {  }
   field_training_format:
-    type: options_buttons
+    type: reliefweb_options
```

```suggestion
    type: options_buttons
```

Not needed for the training format. It's only 2 choices:  online or on site and the description for those terms is basically the term label. It's also not required in the RW-601 ticket.